datafusion/sql/src/statement.rs

```diff
@@ -718,19 +718,13 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
         let mut statement_options = StatementOptions::new(options);
         let file_format = statement_options.try_infer_file_type(&statement.target)?;
-        let single_file_output =
-            statement_options.take_bool_option("single_file_output")?;
```

Comment:
It may be worth keeping the call to take_bool_option and discarding the result. That will allow queries that specify single file output to continue working as expected in most cases.

Reply:
Seems a good idea. 
But I worry that some users didn't specify the output folder path using `/`, but using a 
`single_output_option = false`. Keeping this option may let the query running but has different result. 🤔

Reply:
I agree that it seems cleaner to just remove the option rather than ignore it